recipe/meta.yaml

```diff
@@ -12,19 +12,19 @@ source:
   sha256: {{ hash }}
 
 build:
-  number: 3
-  skip: True  # [osx or linux or py3k]
+  number: 1
+  skip: True  # [osx or linux or py27]
```

Comment:
You can skip `py35` as well because `py35` and `py36` both use the same msvc version